frontend/alert/src/components/managealert/AlertSearch.js

```diff
@@ -25,11 +26,16 @@ const SearchBarFlex = styled(Flex)`
 
 // Component for search filter
 // in alert management
-export const AlertSearch = () => (
+export const AlertSearch = ({ value, onChange }) => (
     <SearchFlex valign>
         <SearchBarFlex valign margin="0.2rem">
             <Img src={Search} alt="" width="0.6rem" height="0.6rem" />
-            <SearchInput type="search" placeholder="Search" />
+            <SearchInput type="search" placeholder="Search" value={value} onChange={onChange} />
```

Comment:
This can be a TODO for another PR to use the `AutoComplete` component from @dlike230.

Reply:
This might be the wrong place for the `AutoComplete` component. I think you might be thinking of `ManageAlertHeader` which is where users actually search for alerts. This search merely lets them filter their own registrations